src/test/java/org/xstefank/check/RequiredRowsCheckTest.java

```diff
@@ -0,0 +1,52 @@
+package org.xstefank.check;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.xstefank.TestUtils;
+import org.xstefank.model.yaml.Row;
+
+import java.io.IOException;
+import java.util.regex.Pattern;
+
+import static java.util.Collections.singletonList;
+
+public class RequiredRowsCheckTest {
+
+    private Row row;
+    private RequiredRowsCheck requiredRowsCheck;
+
+    @Before
+    public void before() {
+        row = new Row();
+        row.setPattern(Pattern.compile("^Test.*description$"));
```

Comment:
I slightly don't understand why it's needed to recompile pattern before each test